src/ImGui.NET/ImGuiNative.cs

```diff
@@ -272,7 +272,7 @@ public static unsafe class ImGuiNative
         public static extern void igTextWrapped(string fmt);
 
         [DllImport(cimguiLib)]
-        public static extern void igTextUnformatted(byte* text, byte* text_end);
+        public static extern void igTextUnformatted(string text, string text_end);
```

Comment:
You're right, I wasn't thinking about the case when one would want to print a substring that doesn't start at the first character. In addition it seems that this is wrong in some other functions as well - igGetIdStrRange, igPushIdStrRange?